docs/pages/by-operation.md

```
@@ -0,0 +1,111 @@
+</script>
+
+<OAOperation :spec="spec" :operationId="operationId" :isDark="isDark" />
+```
+
+</template>
+
+</ScopeConfigurationTabs>
+
+## Example
+
+<SandboxIframe :sandbox-data="{sandboxView: 'preview', previewComponent: 'OAOperation', operationId: 'getAllArtists'}" :iframe-zoom="0.6" class="h-[70vh] max-h-[700px]" />
```

Markdownlint (0.37.0)

Element: SandboxIframe
Inline HTML

(MD033, no-inline-html)